src/Vision.php

```diff
@@ -29,11 +36,26 @@ public function __construct($apiKey, array $features = [])
 
     /**
      * @param Image $image
-     * @return AnnotateImageResponse
+     * @param string $returnType
+     * @return mixed
+     */
+    public function request(Image $image, $responseType = null)
+    {
+        $this->visionRequest = new VisionRequest($this->apiKey, $image, $this->features);
+        $this->visionRequest->send();
+
+        return $this->getResponseType($responseType);
+    }
+
+    /**
+     * @param $responseType
+     * @return mixed|AnnotateImageResponse
      */
-    public function request(Image $image)
+    public function getResponseType($responseType)
```

Comment:
getResponseForType. getResponseType would assume that it returns the response type instead of the output in a specific format.